ROADMAP.md

```diff
@@ -6,44 +6,15 @@ AsyncAPI is a specification for defining message-driven APIs. We're on a mission
 ## How to Get Involved?
 AsyncAPI is an open-source community-driven project. We're happy to receive contributions from different people, no matter your skills or your area of expertise. There are many ways you can contribute to the project, not just code. There is always a need for: documentation, design, writing, evangelizing, code, issue triage, and more! We also welcome financial support and donations. Check out the [contributing guidelines](./CONTRIBUTING.md) to learn more.
 
-## Timeline
+## Versioning
```

Comment:
I don't think we have to explain versioning on a Roadmap document. Also, versioning is already explained on the spec: https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#asyncapi-version-string.

And there's a link to the latest spec in the readme already: https://github.com/asyncapi/asyncapi/#the-specification.

Reply:
Got it. 👍